src/Components/Testimonials/TestimonialsForm.js

```diff
@@ -1,32 +1,96 @@
-import React, { useState } from 'react';
-import '../FormStyles.css';
+import {useEffect, useState} from 'react';
+import {Formik, Form, Field} from 'formik';
+import { ErrorsForm } from '../common/messagesForm';
+import { CKEditor } from '@ckeditor/ckeditor5-react';
+import { UrlInput } from '../common/getUrlInputFile';
+import ClassicEditor from '@ckeditor/ckeditor5-build-classic';
+import { validateFormTestimonials, replaceCKeditor } from '../common/validateFormTestimonials';
```

Comment:
if it goes on common it should go on a validations file because validateFormTestimonials is not too common, its especific